modules/service/src/main/scala/lucuma/odb/logic/Generator.scala

```diff
@@ -259,6 +259,9 @@ object Generator {
           // Commit Hash
           md5.update(commitHash.hashBytes)
 
+          // Completion Declaration
+          md5.update(params.declaredComplete.hashBytes)
+
```

Comment:
I don't know if this is the right approach. The issue is that the execution state query wasn't respecting changes to `c_declared_complete` so I was still getting `Ongoing` back even if the bit was set.

Reply:
D'oh I added the declared complete flag to the `GeneratorParams` hash, but here we're picking individual items out of the params so what you've done is correct.  We may want to switch to `md5.update(params.hashBytes)` and remove the lines that pick individual items out of it.  It would then include the science band which doesn't really impact sequence generation yet, but that also doesn't change often.  It would also include the calibration role, which does matter and probably should have been added here as well.

Reply:
I guess calibration role never changes either, so maybe it's not a big deal. But yeah just to simplify (since caching is hard) I will change it as you suggest.